recipes/tsdownsample/conda_build_config.yaml

```diff
@@ -0,0 +1,2 @@
+channel_sources:
+  - conda-forge/label/rust_dev,conda-forge
```

Comment:
Are you using any dev pkgs below? If not let's remove this.

Reply:
Yeah this doesn't work without using the nightly